MyCryptoHQ/MyCrypto
# Fix Electron Icons & Fix Initial Sizing

### Description

Fixes the icns file, which fixes osx icons. While I was testing other builds, I noticed that linux and windows start with scrollbars, so I upped their initial height a bit too.

### Steps to Test

1. Build for OSX
2. Confirm the icon shows up
